tests/virtual/test_virtual_network.py

```diff
@@ -35,7 +34,7 @@ def connect(dictionary):
     return connect
 
 
-@pytest.mark.virtual
+@pytest.mark.no_connection
```

Comment:
Why you change that?

Reply:
Because they are wrong, the correct marker for using the virtual drive is `no_connection`